src/System.Device.Gpio.Tests/RaspberryPiComputeModuleDriverTests.Linux.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.Gpio.Drivers;
+using System.Threading;
+using Xunit;
+
+namespace System.Device.Gpio.Tests
+{
+    public class RaspberryPiComputeModuleDriverTests : GpioControllerTestBase
```

Comment:
Adding System.Device.Gpio tests is a bit complex since we actually run these on actual hardware so we rely on things being connected in the right way in order for tests to be able to run correctly. Because of that reason, let's hold off on adding any tests for this for now, and let's first get the implementation working, and then we can add the tests later after connecting the devices properly in the lab.